go/vt/sqlparser/sql.y

```diff
@@ -4316,6 +4419,7 @@ reserved_keyword:
 | LEFT
 | LIKE
 | LIMIT
+| LOCAL
```

Comment:
Does this need to go here? Can it go in non_reserved? The only reason to put it here is if it causes conflicts in the grammar otherwise.